### PR TITLE
add(uiVue): add to packages

### PR DESCRIPTION
**Pull request by Sicorschi-Tudor**
https://github.com/bagrinsergiu/blox-editor/issues/23324

### CODE REVIEW

File: packages/tsconfig/vue-library.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "$schema": "https://json.schemastore.org/tsconfig",
+  "display": "Vue Library",
+  "extends": "@vue/tsconfig/tsconfig.json",
+  "compilerOptions": {
+    "declaration": true,
+    "declarationMap": false,
+    "ignoreDeprecations": "5.0"
```
Comment (maxval1) [inline packages/tsconfig/vue-library.json:8]: vezi daca mai trebue `"ignoreDeprecations": "5.0"` in ts 5 nu mai trebue din cit imi aduc aminte

File: packages/ui-vue/package.json [hunk 2]
```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@brizy/ui-vue",
+  "private": true,
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "scripts": {
+    "lint": "eslint \"src/**/*.ts*\"",
+    "tsc": "vue-tsc --noEmit",
+    "dev": "npm run clean && npm run build:types && webpack --watch",
```
Comment (maxval1) [inline packages/ui-vue/package.json:9]: missing `prettier-check`

File: packages/ui-vue/tailwind.config.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+/** @type {import('tailwindcss').Config} */
+module.exports = {
```
Comment (maxval1) [inline packages/ui-vue/tailwind.config.js:2]: pt ce trebue tailwind ?

File: packages/ui-vue/postcss.config.js [hunk 4]
```diff
@@ -0,0 +1,6 @@
+module.exports = {
```
Comment (maxval1) [inline packages/ui-vue/postcss.config.js:1]: pt ce trebue postcss?

File: packages/ui-vue/webpack.config.ts [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+      {
+        test: /\.css$/i,
+        include: path.resolve(__dirname, "src"),
+        use: [MiniCssExtractPlugin.loader, "css-loader", "postcss-loader"],
+      },
+      {
+        test: /\.(svg)$/,
+        type: "asset/inline",
+      },
+    ],
```
Comment (maxval1) [inline packages/ui-vue/webpack.config.ts:40]: nu sigur ca e nevoie de loaders de css si svg

**Review by maxval1 — CHANGES_REQUESTED** [review]

**Review by maxval1 — CHANGES_REQUESTED** [review]
de facut un rename din ui-vue in vue

File: packages/ui-vue/webpack.config.ts (hunk 5 above)
Comment (Sicorschi-Tudor) [reply packages/ui-vue/webpack.config.ts:40]: Screenshot from 2023-07-03 11-28-32

File: packages/tsconfig/vue-library.json (hunk 1 above)
Comment (Sicorschi-Tudor) [reply packages/tsconfig/vue-library.json:8]: Screenshot from 2023-07-03 11-28-32

**Review by maxval1 — CHANGES_REQUESTED** [review]

**Pull request merged**